src/charts/utilization-bar/utilization-bar-chart.component.js

```diff
@@ -185,5 +193,10 @@ angular.module('patternfly.charts').component('pfUtilizationBarChart', {
         ctrl.updateAll();
       }
     };
+
+    ctrl.tooltipMessage = function (type) {
+      return ctrl.tooltipFunction ? ctrl.tooltipFunction() : type === 'used' ? (100 - ctrl.chartData.percentageUsed)
+        + '% Available' : ctrl.chartData.percentageUsed + '% Used';
+    };
```

Comment:
I'd rather see the message creation in a separate function to make the read more easily:

```return ctrl.tooltipFunction ? || ctrl.tooltipFunction()  : defaultTooltipFunction(type);```